bin/resty

```diff
@@ -130,7 +130,15 @@ my $conns = $conns_num || 64;
 my @nameservers;
 
 if (!$resolve_ipv6) {
-    unshift @nameservers, "ipv6=off";
+    # no error check to let the later exec() fail.
+    # worst case: ipv6=off is not added to the nginx
+    # conf, but the worker won't spawn anyways...
+    my $stderr = `$nginx_path -V 2>&1 1>/dev/null`;
+    #warn $stderr;
+
+    if (index($stderr, "--with-ipv6") != -1) {
```

Comment:
> For nginx 1.11.5+, the behavior is not really what we are looking for when the operating system supports IPv6 but the network is actually not :)

I agree, but if you followed the thread on the nginx-devel mailing list (it seems you have), I do not know how to address this any other way anymore :D

> Will propose a patch for the nginx core to the openresty/openresty repo to silently ignore the ipv6=off directive argument when IPv6 is not enabled?

Yes, I will propose the same patch as the one proposed to the nginx-devel mailing list.